### DISCUSSION TITLE
Bulk editing of games

### DISCUSSION THREAD

**Discussion by benbarden** [benbarden/switchscores#96] [Ideas]
I've built a new feature that might come in handy - a way to bulk edit games.

I'm using it here - https://www.switchscores.com/staff/games/list/upcoming-eshop-crosscheck

Click "Bulk update order" and you'll see what it does. (please don't change the numbers) 😄 

The idea is that each "list" will load different games and show a field to be edited. I can add more lists.

There's also a custom edit mode, which currently only supports editing categories:

https://www.switchscores.com/staff/games/bulk-edit/edit-game-id-list/category/1,2,3,4,5

Change the numbers to whichever game IDs you want to list. This wouldn't be something you'd do manually, but I could link it from a page of search results.

Is there anything else that this could be used for? Unfortunately, editing linked data such as publishers and tags wouldn't be easy to do in bulk, as those can have multiple values. But single value fields could work.

**Comment by cosmo0** [comment]
Tags! :) As I said in #95 I'd love to be able to tag multiple games at once with the same tags. It wouldn't list tags for each game, but apply the same tags to all of the selected games.

**Reply by benbarden** [reply]
Unfortunately, as I said in the post, editing linked data wouldn't be easy to do in bulk - at least not on this screen. I'll have to think about other ways to handle tags, but it would be separate from this screen.
